Core/src/org/sleuthkit/autopsy/corecomponentinterfaces/DataContentViewer.java

```diff
@@ -27,6 +27,10 @@
  * some form of JPanel. We find it easiest to use the NetBeans IDE to first make
  * a "JPanel Form" class and then have it implement DataContentViewer. This
  * allows you to easily use the UI builder for the layout.
+ * 
+ * DataContentViewer panels should handle their own vertical scrolling, the horizontal 
+ * scrolling when under their panel's preffered size will be handled by the DataContentPanel
```

Comment:
Please correct "prefferred" to "preferred."

In terms of documentation, please take a look at ~\autopsy\docs\doxygen\modContent.dox and consider adding some documentation there as well, thanks!